### PR TITLE
Ask CFPB Wagtail editor changes

### PR DESCRIPTION
**Pull request by csebianlander**
This PR makes a variety of changes to the Wagtail user experience for Answer pages, simplifying the editing process for Ask CFPB as a whole.

---

## Removals
- "Statement" field
- "Related resource" block (and the entire snippet, as this was its only use-case)
- "Featured?" checkbox
- Various Sidebar block options

## Changes
- Many field labels and helper texts across the Answer page type

## How to test this PR

1. localhost
2. Attempt to edit any "Answer page" type

## Screenshots
Not including only because they'd have to be so long as to be unreadable; better to pull up a side-by-side of the current environment and your local when reviewing this PR.

## Notes and todos

- The content for the Ask CFPB landing page is converted to static HTML to support the immediate removal of the "Featured?" checkbox on the Answer page type. Future work will rebuild this element as a block on the Answer landing page instead.

## Checklist

- [ ] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### CODE REVIEW

**Comment by chosak** [comment]
Do these changes let us fully retire the RelatedResource snippet type? Is that content getting migrated somewhere else in the answer?

File: cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html [hunk 1]
```diff
@@ -49,50 +49,791 @@ <h2 class="u-show-on-mobile u-mt45">
 
             <div class="u-hide-on-mobile question-categories u-mb15">
 
-                {#
-                    TODO: This can use the standard card-group.html template
-                          if the partially styled links are removed.
```
Comment (anselmbradford) [inline cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:54]: Why was this deleted? I still see the `a-link__partially-styled` class below.

File: cfgov/ask_cfpb/jinja2/ask-cfpb/answer-page.html [hunk 2]
```diff
@@ -63,17 +63,6 @@ <h1>
                 </button>
             </div>
 
-            {% if page.related_resource and page.related_resource.trans_text(page.language) | striptags %}
-            <div class="block next-steps">
```
Comment (anselmbradford) [inline cfgov/ask_cfpb/jinja2/ask-cfpb/answer-page.html:67]: It looks like you could remove the `.next-steps` class from the CSS in the ask less file at https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/on-demand/ask.less#L187

I'm not seeing another reference to next-steps in Ask (though it does appear in other apps, but they should have their own CSS for it).

File: cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html [hunk 3]
```diff
@@ -49,50 +49,789 @@ <h2 class="u-show-on-mobile u-mt45">
...
+                                <a class="" href="/es/herramientas-del-consumidor/prestamos-para-vehiculos/">
+                                    {{ svg_icon( "car" ) }}
+                                    Préstamos para vehículos
+                                </a>
+                            </h2>
+
+                            <ul class="m-list">
+                                
+                                    <li class="m-list_item">
+                                    <a href="/es/obtener-respuestas/que-efecto-tendria-sobre-mi-credito-comparar-diferentes-ofertas-de-prestamos-para-vehiculos-es-763/" class="a_link a-link__partially-styled">
```
Comment (anselmbradford) [inline cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:488]: Do the Spanish pages not have a `data-pretty-href` URL, like the english pages do? This should only come into play when printing.

File: cfgov/ask_cfpb/models/answer_page.py [hunk 4]
```diff
@@ -109,21 +109,19 @@ class AnswerPage(CFGOVPage):
...
-            "(Optional) Use this field to rephrase the question title as "
-            "a statement. Use only if this answer has been chosen to appear "
-            "on a money topic portal (e.g. /consumer-tools/debt-collection)."
+            "If content is changed, update this "
+            "to match the new publication date."
         ),
     )
+    question = models.TextField(
+        blank=True,
+        help_text=("Used as the primary heading (H1) " "of the page."),
```
Comment (anselmbradford) [inline cfgov/ask_cfpb/models/answer_page.py:119]: Do we need the `" "` break in this?

File: cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html (hunk 3 above)
Comment (csebianlander) [reply cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:488]: That's actually a good question -- I don't think they do, just based on the fact that I pulled this HTML from the current rendered pages in both cases. Also not seeing anything that would work for Spanish in `redirects.conf` though I might just be missing it...

Comment (csebianlander) [reply cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:488]: just scanned through a few live pages to compare Spanish and English, and yeah, there's no equivalent shortening for `data-pretty-href` in Spanish.

File: cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html [hunk 5]
```diff
@@ -49,50 +49,789 @@ <h2 class="u-show-on-mobile u-mt45">
...
+                                <a class="" href="/consumer-tools/auto-loans/">
+                                    {{ svg_icon( "car" ) }}
+                                    <span>Auto loans</span>
+                                </a>
+                            </h2>
+
+                            <ul class="m-list">
+                                
+                                    <li class="m-list_item">
+                                    <a class="a_link a-link__partially-styled" data-pretty-href="cfpb.gov/askcfpb/763" href="/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/">
```
Comment (chosak) [inline cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:73]: Per my other comment just now about `data-pretty-href`, these should be attended automatically by the Django middleware, so it would be better not to add them manually in the template.

File: cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html (hunk 3 above)
Comment (csebianlander) [reply cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:488]: Yeah there's no equivalent pattern to Spanish.

File: cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html [hunk 6]
```diff
@@ -49,50 +49,789 @@ <h2 class="u-show-on-mobile u-mt45">
...
+                                </a>
+                            </h2>
+
+                            <ul class="m-list">
+                                
+                                    <li class="m-list_item">
+                                    <a class="a_link a-link__partially-styled" href="/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/">
+<span class="a-link__partially-styled_plain">What effect will shopping for an auto loan have on my credit? </span>
+<span class="a-link__partially-styled_underlined">Read answer</span>
+</a>
```
Comment (chosak) [inline cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:76]: I note your comment about this hardcoding being only temporarily; if so, what do you think about implementing this in Python instead via a hardcoded list of Answer IDs. See implementation [here](https://github.com/cfpb/consumerfinance.gov/commit/c13a3d54b0ddc005c739a8b4276546215f4f71a0). This lets you leave the landing-page.html as-is before your PR, without duplicating the various boilerplate for each link.

Comment (csebianlander) [reply cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html:76]: Love it -- merging these changes!

**Comment by csebianlander** [comment]
Andy's Python hard-coding approach had the benefit of making it very easy to see all the discrepancies between the English and Spanish landing pages -- since I know for a fact the Spanish is out of sync, I've updated that segment so that they match.

**Review by chosak — APPROVED** [review]
